treeherder/perf/models.py

```diff
@@ -40,7 +40,7 @@ class PerformanceSignature(models.Model):
     option_collection = models.ForeignKey(OptionCollection, on_delete=models.CASCADE)
     suite = models.CharField(max_length=80)
     test = models.CharField(max_length=80, blank=True)
-    application = models.CharField(max_length=10, null=True,
+    application = models.CharField(max_length=10, default='', blank=True,
```

Comment:
The `application` field will only be provided via the background workers.
`blank=True` targets something entirely different; it's mostly concerned with forms originating from the frontend.

Reply:
Done.